core/src/banking_stage/committer.rs

```diff
@@ -34,7 +34,6 @@ impl Committer {
         batch: &TransactionBatch,
         loaded_transactions: &mut [TransactionLoadResult],
         execution_results: Vec<TransactionExecutionResult>,
-        sanitized_txs: &[SanitizedTransaction],
```

Comment:
Good to have lesser arguments

Reply:
^ yeah, already passing the batch which contains the sanitized transactions